app/views/concerns/facet_entry_point_displaying_view.rb

```diff
@@ -6,47 +6,14 @@ module FacetEntryPointDisplayingView
 
   protected
 
-  def facet_entry_items(facet_link_group, page = nil)
-    facet_link_group.browse_entry_facet_entries.map do |entry_point|
-      thumb_url = Rails.cache.fetch("facet_link_groups/#{facet_link_group.id}/#{entry_point.id}/thumbnail_url")
-      facet_entry_item(entry_point, thumb_url, page)
-    end
-  end
-
-  def facet_entry_item(entry, thumb_url = nil, page = nil)
-    {
-      url: browse_entry_path(entry, page),
-      label: entry.title,
-      image_url: thumb_url,
-      image_alt: nil
-    }
-    # Use this model to behave more like normal browse entries,
-    # however this needs frontend alignment.
-    # {
-    #     title: entry.title,
-    #     url: browse_entry_path(entry, page),
-    #     image: entry.file.nil? ? nil : entry.file.url,
-    #     image_alt: nil
-    # }
-  end
-
   ##
   # @param page [Page]
   def facet_entry_items_grouped(page)
```

Comment:
@lbiedinger this could be made more concise like:
```ruby
  def facet_entry_items_grouped(page)
    page.facet_link_groups.map do |facet_link_group|
      FacetLinkGroupPresenter.new(facet_link_group, controller, blacklight_config, page).display
    end
  end
```